device: cuda:5
output directory: RESULTS/cifar10-1k/fc-relu-depth4/seed_0/mse/gd//lr_0.8
Files already downloaded and verified
Files already downloaded and verified
eigenvalues:  tensor([2.0154, 1.9996])
Traceback (most recent call last):
  File "src/gd.py", line 126, in <module>
    seed=args.seed,device_id=args.device_id)
  File "src/gd.py", line 58, in main
    store_singular_values(network, singular_value_storage)
  File "/egr/research-slim/ghoshavr/edge-of-stability/src/utilities.py", line 242, in store_singular_values
    u, s, v = torch.svd(layer.weight.data)
RuntimeError: cusolver error: CUSOLVER_STATUS_INTERNAL_ERROR, when calling `cusolverDnCreate(handle)`
